driver/src/main/java/org/neo4j/driver/internal/cluster/ClusterRoutingTable.java

```diff
@@ -78,9 +78,7 @@ public boolean isStale()
     @Override
     public synchronized void forget( BoltServerAddress address )
     {
-        // Don't remove it from the set of routers, since that might mean we lose our ability to re-discover,
-        // just remove it from the set of readers and writers, so that we don't use it for actual work without
-        // performing discovery first.
+        routers.remove( address );
```

Comment:
What's the point to keep in routing table? they were there to protect from running out of routers. Now we no longer have the problem with dns